README.md

```diff
@@ -78,7 +78,8 @@ Then add ZACS to your Babel config (`.babelrc` or `babel.config.js`):
   "plugins": [
 +   ["@nozbe/zacs/babel", {
 +     "platform": "web", // or "native"
-+     "production": true // pass `false` to enable debug attributes
++     "production": true, // pass `false` to enable debug attributes
++     "keepDeclarations": false // pass `true` to keep zacs.xxx variable declarations in output
```

Comment:
Are all properties required? Maybe it would be a good idea to highlight all optional props and their default values?

Reply:
nope, not required - but so what?

Reply:
To indicate what's important and what can be configured later. It might be helpful for newcomers fiddling with the lib